Intersect.Editor/Localization/Strings.cs

```diff
@@ -2042,6 +2047,10 @@ public struct EventConditional
 
             public static LocalizedString canstartquest = @"Can Start Quest";
 
+            public static LocalizedString checkequippedslotlbl = @"Slot";
+
+            public static LocalizedString checkequippedslotgrp = @"Check Item Slot:";
```

Comment:
Name the string `Check Equipment` and change the text to `Check Equipment`, do not include `:` in the string.